ArduCopter/config.h

```diff
@@ -141,7 +141,12 @@
 //////////////////////////////////////////////////////////////////////////////
 // Precision Landing with companion computer or IRLock sensor
 #ifndef PRECISION_LANDING
- # define PRECISION_LANDING ENABLED
+  #ifndef AP_PREC_LAND_ENABLED
+    #define AP_PREC_LAND_ENABLED ENABLED
+    #define PRECISION_LANDING ENABLED
+  #else
+    #define PRECISION_LANDING AP_PREC_LAND_ENABLED
+  #endif
```

Comment:
```suggestion
```

... just remove `PRECISION_LANDING` from Copter entirely - Randy has asked for things to be this way on previous PRs.

Reply:
.... to be clear, replace it with `AC_PRECLAND_ENABLED` :-)